HLTrigger/Configuration/python/deepTauAtHLT.py

```diff
@@ -0,0 +1,357 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoTauTag.RecoTau.pfRecoTauDiscriminationByIsolation_cfi import *
+from RecoTauTag.RecoTau.PFRecoTauQualityCuts_cfi import PFTauQualityCuts
+
+
+from RecoTauTag.RecoTau.PFTauPrimaryVertexProducer_cfi      import *
+from RecoTauTag.RecoTau.PFTauSecondaryVertexProducer_cfi    import *
+from RecoTauTag.RecoTau.PFTauTransverseImpactParameters_cfi import *
+
+
+def update(process):
+    process.options.wantSummary = cms.untracked.bool(True)
+
+    process.hltFixedGridRhoFastjetAll = cms.EDProducer( "FixedGridRhoProducerFastjet",
+        gridSpacing = cms.double( 0.55 ),
+        maxRapidity = cms.double( 5.0 ),
+        pfCandidatesTag = cms.InputTag( "hltParticleFlowReg" )
+    )
+
+    PFTauQualityCuts.primaryVertexSrc = cms.InputTag("hltPixelVertices")
+
+    process.chargedIsoPtSum = pfRecoTauDiscriminationByIsolation.clone(
```

Comment:
I think it would be possible to use the new container format here to produce and store these isolation quantities in one go. Did you consider this? I did not check in detail, but probably you would only need two instances for the two dR versions. (see for example https://github.com/cms-sw/cmssw/blob/master/RecoTauTag/Configuration/python/HPSPFTaus_cff.py#L68-L168)

Reply:
It seems I was a bit too fast, I made a typo somewhere, but now I got the new container format working